game/views.py

```
@@ -592,7 +600,29 @@ def ai_move(request):
     depth_map = {'easy': 1, 'medium': 2, 'hard': 3}
     depth = depth_map.get(difficulty, 2)
 
-    best = game.get_ai_move(depth=depth)
+    opening = request.session.get('opening', '')
+    book_move = None
+    if opening:
+        try:
+            ai_half_moves = len(game.move_history)
+            book_move = get_opening_reply(opening, ai_half_moves)
+        except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)